x-pack/plugins/monitoring/server/kibana_monitoring/collectors/get_ops_stats_collector.js

```diff
@@ -6,47 +6,48 @@
 
 import { KIBANA_STATS_TYPE } from '../../../common/constants';
 import { opsBuffer } from './ops_buffer';
-import { Collector } from '../classes/collector';
+import { Collector } from '../classes';
+
+let monitor = null; // singleton because start and stop can be called repeatedly because of status event repetitions
```

Comment:
Module scope is always a problem, especially when it's accessed in a "per server" way. People who are running multiple servers in the same process will have the monitor variable clobbered. Sounds like there is in issue causing start() to be called multiple times that should be fixed instead of storing this in module scope.

Reply:
> Sounds like the real issue is that start() is being called multiple times when it's not supposed to be?

That only happens in the unit tests. I need put better handling in the tests, probably by going with the advice from @pickypg

Reply:
Oops, for some reason, I thought this comment was about the timer of the bulk uploader